lib/digital_scriptorium/ds_meta.rb

```diff
@@ -21,12 +21,8 @@ def initialize(record, export_hash)
       @record = record
     end
 
-    def current?(holding)
-      holding.holding_status == HOLDING_STATUS_CURRENT
-    end
-
     def current_holdings(manuscript, export_hash)
-      manuscript.holding_ids.filter_map { |id| export_hash[id] if current?(export_hash[id]) }
+      manuscript.holding_ids.filter_map { |id| export_hash[id] if export_hash[id].current? }
```

Comment:
It shouldn't happened, but it is possible for a Manuscript to point to a deleted Holding, in which case `export_hash[id]` could be `nil`.

```suggestion
      manuscript.holding_ids.filter_map { |id| export_hash[id] if export_hash[id]&.current? }
```